src/PowerShellEditorServices/Services/Workspace/LanguageServerSettings.cs

```diff
@@ -250,59 +258,62 @@ public CodeFormattingSettings(CodeFormattingSettings codeFormattingSettings)
-                    }},
-                    {"PSAlignAssignmentStatement", new Hashtable {
-                        {"Enable", true},
-                        {"CheckHashtable", AlignPropertyValuePairs}
-                    }},
-                    {"PSUseCorrectCasing", new Hashtable {
-                        {"Enable", UseCorrectCasing}
-                    }},
-                };
+            var ruleConfigurations = new Hashtable
+            {
+                { "PSPlaceOpenBrace", new Hashtable {
+                    { "Enable", true },
+                    { "OnSameLine", OpenBraceOnSameLine },
+                    { "NewLineAfter", NewLineAfterOpenBrace },
+                    { "IgnoreOneLineBlock", IgnoreOneLineBlock }
+                }},
+                { "PSPlaceCloseBrace", new Hashtable {
+                    { "Enable", true },
+                    { "NewLineAfter", NewLineAfterCloseBrace },
+                    { "IgnoreOneLineBlock", IgnoreOneLineBlock }
+                }},
+                { "PSUseConsistentIndentation", new Hashtable {
+                    { "Enable", true },
+                    { "IndentationSize", tabSize },
+                    { "PipelineIndentation", PipelineIndentationStyle },
+                    { "Kind", insertSpaces ? "space" : "tab" }
+                }},
+                { "PSUseConsistentWhitespace", new Hashtable {
+                    { "Enable", true },
+                    { "CheckOpenBrace", WhitespaceBeforeOpenBrace },
+                    { "CheckOpenParen", WhitespaceBeforeOpenParen },
+                    { "CheckOperator", WhitespaceAroundOperator },
+                    { "CheckSeparator", WhitespaceAfterSeparator },
+                    { "CheckInnerBrace", WhitespaceInsideBrace },
+                    { "CheckPipe", WhitespaceAroundPipe },
+                }},
+                { "PSAlignAssignmentStatement", new Hashtable {
+                    { "Enable", true },
+                    { "CheckHashtable", AlignPropertyValuePairs }
+                }},
+                { "PSUseCorrectCasing", new Hashtable {
+                    { "Enable", UseCorrectCasing }
+                }},
+            };
+
             if (AutoCorrectAliases)
             {
+                // Empty hashtable required to activate the rule,
+                // since PSAvoidUsingCmdletAliases inherits from IScriptRule and not ConfigurableRule
                 ruleConfigurations.Add("PSAvoidUsingCmdletAliases", new Hashtable());
             }
 
-            return new Hashtable
+            return new Hashtable()
             {
-                {"IncludeRules", new string[] {
-                     "PSPlaceCloseBrace",
-                     "PSPlaceOpenBrace",
-                     "PSUseConsistentWhitespace",
-                     "PSUseConsistentIndentation",
-                     "PSAlignAssignmentStatement",
-                     "PSAvoidUsingCmdletAliases",
```

Comment:
@TylerLeonhardt This is a regression that means the preview extension can not correct aliases any more via the `powershell.codeFormatting.autoCorrectAliases` setting. Will open a PR

Reply:
sorry, I was wrong, forget that. I'll delete that comment